ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -286,7 +293,7 @@ openshift_client_tarball  = "https://mirror.openshift.com/pub/openshift-v4/multi
 release_image_override    = "$(openshift-install version | grep 'release image' | awk '{print $3}')"
 use_zone_info_for_names   = true
 use_ibm_cloud_services    = true
-ibm_cloud_vpc_name        = "${WORKSPACE_NAME}-vpc"
+ibm_cloud_vpc_name        = "${VPC_NAME}"
```

Comment:
consistent with 265

Reply:
Yes, it will help to other to understand the proper variable name been used .